locations/spiders/brookdale.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import json
+from locations.items import GeojsonPointItem
+
+URL = 'https://www.brookdale.com/bin/brookdale/community-search?care_type_category=resident&loc=&finrpt=&state='
+
+US_STATES = (
+       "AL", "AZ", "AR", "CA", "CO", "CT", "DE", "FL", "GA",
+       "ID", "IL", "IN", "IA", "KS", "KY", "LA", "MD",
+       "MA", "MI", "MN", "MS", "MO", "MT", "NE", "NV", "NH", "NJ",
+       "NM", "NY", "NC", "OH", "OK", "OR", "PA", "RI", "SC",
+       "TN", "TX", "UT", "VT", "VA", "WA", "WV", "WI", "WY",
+)
+
+class TemplateSpider(scrapy.Spider):
+    name = "brookdale"
+    allowed_domains = ["www.brookdale.com"]
+
+    def start_requests(self):
+        for state in US_STATES:
+            url = ''.join([URL, state])
+            yield scrapy.Request(url, callback=self.parse_info)
+
+
+    def parse_info(self, response):
+
+        data = json.loads(response.body_as_unicode())
+
+        i = 0
+        while i < len(data):
```

Comment:
It looks like you could just use a for loop here instead of a while loop. For example:

```python
for row in data:
  properties = { ... }
  yield GeojsonPointItem(**properties)
```